recipe/meta.yaml

```diff
@@ -28,22 +30,30 @@ test:
     - qg3-08.cnf
     - uf20-098.cnf
     - sudoku.py
-
+    - test_package.py
+  requires:
+    - pip
   imports:
     - pycosat
+  commands:
+    - pip check
+    - python test_package.py "{{ version }}"
 
 about:
-  home: https://github.com/ContinuumIO/pycosat
+  home: https://github.com/conda/pycosat
   license: MIT
+  license_file:
+    - LICENSE
+    - COPYRIGHT
```

Comment:
Only LICENSE file is available https://github.com/conda/pycosat/blob/0.6.4/LICENSE